pegasus/sites.v3/code.org/views/curriculum_pathway.haml

```diff
@@ -0,0 +1,75 @@
+= inline_css "/generated/curriculum-pathway.css"
+
+%figure.curriculum-pathway.desktop
+  .header
+    .school.elementary.border-bottom-primary-light
+      %p=hoc_s(:grade_level_label_elementary)
+    .school.middle.border-bottom-primary-medium
+      %p=hoc_s(:grade_level_label_middle)
+    .school.high.border-bottom-primary-default
+      %p=hoc_s(:grade_level_label_high)
+    .grade
+      %p K
+    - (1..12).each do |grade|
+      .grade
+        %p= grade
+
+  .courses
+    %a.course.csf.border-top-primary-light{href: "/csf"}
+      %p=hoc_s(:course_name_csf)
+    %a.course.csc.border-top-primary-light{href: "/csc"}
+      %p=hoc_s(:course_name_csc)
+    %a.course.csd.border-top-primary-medium{href: "/csd"}
+      %p=hoc_s(:course_name_csd)
+    %a.course.csp.border-top-primary-default{href: "/csp"}
+      %p=hoc_s(:course_name_csp)
+    %a.course.csa.border-top-primary-default{href: "/csa"}
+      %p=hoc_s(:course_name_csa)
+
+  .background
+    - (1..13).each do |i|
+      %div
+
+%figure.curriculum-pathway.mobile
+  .school-wrapper
```

Comment:
I tried using helper methods for all of these but kept getting syntax errors — going with the long version in an effort to get this out 🥲